lib/aurora_bootstrapper/s3file.rb

```diff
@@ -0,0 +1,49 @@
+module AuroraBootstrapper
+    class S3file
```

Comment:
is this used anywhere?

Reply:
I do not think so.

Reply:
should we delete it? i think that the `File` class was accidentally left as an oversight from a previous version. If this is a rename, we should just get rid of it altogether

Reply:
ok